src/NuGet.Clients/NuGet.SolutionRestoreManager/VsSolutionRestoreService.cs

```diff
@@ -76,6 +77,21 @@ public sealed class VsSolutionRestoreService : IVsSolutionRestoreService
 
         public Task<bool> CurrentRestoreOperation => _restoreWorker.CurrentRestoreOperation;
 
+        public Task<bool> NominateProjectAsync(string projectUniqueName, CancellationToken token)
```

Comment:
Its not really needed. We decided to keep it so that in when/if we add project restore only option it can be extended for it.